integrations/terraform/gen/main.go

```diff
@@ -601,6 +601,32 @@ var (
 		ForceSetKind: "apitypes.KindAutoUpdateConfig",
 		DefaultName:  "apitypes.MetaNameAutoUpdateConfig",
 	}
+
+	healthCheckConfig = payload{
+		Name:                  "HealthCheckConfig",
+		TypeName:              "HealthCheckConfig",
+		VarName:               "healthCheckConfig",
+		GetMethod:             "GetHealthCheckConfig",
+		CreateMethod:          "CreateHealthCheckConfig",
+		UpsertMethodArity:     2,
+		UpdateMethod:          "UpdateHealthCheckConfig",
```

Comment:
Is this service doing unconditional or conditional updates? If it's the latter, you will want to use upsert instead of update.